tests/amazon/aws/hooks/test_emr.py

```diff
@@ -14,6 +15,13 @@
 AWS_CONN_ID = "aws_default"
 JOB_FLOW_ID = "j-T0CT8Z0C20NT"
 STEP_ID = "s-34RJO0CKERRPL"
+JOB_ROLE_ARN = os.getenv("JOB_ROLE_ARN", "arn:aws:iam::012345678912:role/emr_eks_default_role")
+JOB_NAME = "JOB_NAME"
+EXECUTION_ROLE_ARN = JOB_ROLE_ARN
+RELEASE_LABEL = ""
```

Comment:
why is the RELEASE_LABEL set to empty string?

Reply:
Removed this since I have moved submit_job to the operator so not require here